extras/rya.geoindexing/geo.common/src/main/java/org/apache/rya/indexing/GeoIndexerType.java

```diff
@@ -31,31 +28,49 @@
     /**
      * Geo Mesa based indexer.
      */
-    GEO_MESA(GeoMesaGeoIndexer.class),
+    GEO_MESA("org.apache.rya.indexing.accumulo.geo.GeoMesaGeoIndexer"),
     /**
      * Geo Wave based indexer.
      */
-    GEO_WAVE(GeoWaveGeoIndexer.class),
+    GEO_WAVE("org.apache.rya.indexing.accumulo.geo.GeoWaveGeoIndexer"),
     /**
      * MongoDB based indexer.
      */
-    MONGO_DB(MongoGeoIndexer.class);
+    MONGO_DB("org.apache.rya.indexing.mongodb.geo.MongoGeoIndexer"),
+	
+	MONGO_GEO_TEMPORAL("org.apache.rya.indexing.geotemporal.mongo.MongoGeoTemporalIndexer"),
+	MONGO_GEO_TEMPORAL_OPTIMIZER("org.apache.rya.indexing.geotemporal.GeoTemporalOptimizer");
```

Comment:
the MongoGeoTemporal indexer is not a geoindexer, at least not as far as this class should be concerned